typesafety/test_interfaces/test_specific/test_ioresult/test_ioresultlike_inheritance.yml

```diff
@@ -159,7 +159,16 @@
     main:18: note: See https://mypy.readthedocs.io/en/stable/common_issues.html#incompatible-overrides
     main:18: note: This violates the Liskov substitution principle
     main:27: error: Signature of "from_ioresult" incompatible with supertype "IOResultLikeN"
+    main:27: note:          def [_NewValueType, _NewErrorType] from_ioresult(self, inner_value: IOResult[_NewValueType, _NewErrorType]) -> MyClass[_NewValueType, _NewErrorType]
+    main:27: note:          def [_ValueType, _ErrorType] from_ioresult(cls, inner_value: IOResult[_ValueType, _ErrorType]) -> KindN[MyClass[_ValueType, _ErrorType], _ValueType, _ErrorType, NoReturn]
+    main:27: note:      Subclass:
```

Comment:
Can you please also bump `pytest-mypy-plugins`? This error will go away (I mean invalid ordering).

Reply:
sure